dev/com.ibm.ws.jaxws.2.3.common/bnd.bnd

```diff
@@ -51,6 +51,7 @@ Import-Package: \
    !org.apache.tools.ant;version=!, \
    !com.ibm.ws.jaxws.globalhandler, \
    !com.ibm.ws.jaxws.threading, \
+   !com.sun.tools.ws.ant, \
```

Comment:
Not sure why this bundle was importing this package, when it's only referenced in the `com.ibm.ws.jaxws.tools` bundle.